packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-headers.jsx

```diff
@@ -5,9 +5,31 @@ import {
     VerticalTabHeaderTitle }  from './vertical-tab-header.jsx';
 
 class VerticalTabHeaders extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
+        this.headers = [];
+        this.state = { top: 0 };
+    }
+
+    componentDidMount() {
+        this.headers = this.ref.current.querySelectorAll('.vertical-tab__header__link');
+        this.setState({ top: this.offsetTop });
+    }
+
+    componentDidUpdate() {
+        this.setState({ top: this.offsetTop });
+    }
+
+    get offsetTop() {
```

Comment:
Please use snake case for getters, `offset_top`. Also, since you are setting `state.top` from the computed property, and not mutating it any further, I believe it is safe to remove `state.top` altogether. Refer below comment.

Reply:
thanks, good point 👍